tasks/calamari_nosetests.py

```diff
@@ -113,10 +114,17 @@ def clone_calamari(config, client):
     branch = config.get('calamari_branch', 'master')
     url = config.get('calamari_giturl', 'git://github.com/ceph/calamari')
     try:
-        cmd = 'git clone -b {branch} {giturl}'.format(
-            branch=branch, giturl=url
+        out = StringIO()
+        # ensure branch is present (clone -b will succeed even if
+        # the branch doesn't exist, falling back to master)
+        proc = client.run(
```

Comment:
You've got a flake8 error here: "local variable 'proc' is assigned to but never used"